config.toml

```diff
@@ -92,24 +92,58 @@ description = """
  parent = "about"
  url = "https://letsencrypt.status.io/"
 
+[languages.zh-cn]
+title = "Let's Encrypt - 免费SSL/TLS证书"
+languageName = "简体中文"
+contentDir = "content/zh-cn"
+languageCode= "zh-cn"
+weight = 1
+description = """
+  Let's Encrypt 是由非营利性互联网安全研究组（ISRG）提供给您的免费，自动化和开放的证书颁发机构。
+  """
+[[languages.zh-cn.menu.main]]
+  name = "获取帮助"
+  weight = 20
+  url = "https://community.letsencrypt.org/"
+[[languages.zh-cn.menu.main]]
+  name = "捐赠"
+  weight = 30
+  identifier = "donate"
+  url = "/zh-cn/donate/"
+[[languages.zh-cn.menu.main]]
+  name = "关于我们"
+  weight = 40
+  identifier = "about"
+  url = "/zh-cn/about/"
+[[languages.zh-cn.menu.main]]
+  name = "服务状态"
+  weight = 60
+  identifier = "status"
+  parent = "about"
+  url = "https://letsencrypt.status.io/"
+
 #[languages.no]
-#title = "Let's Encrypt - gratis SSL/TLS-sertifikat"
-#description = "Let&rsquo;s&nbsp;Encrypt er ein gratis og ..."
-#languageName ="Norsk"
+#title = "Let's Encrypt - Free SSL/TLS Certificates"
```

Comment:
Why those changes for the language "no"?

Reply:
Because those languages are in French?? 

Probably should made it into

Reply:
It was in "Norsk" ("no"), but just the beginning of a translation, but now there is also French as an example, maybe these comments can be removed.